client/consensus/aura/src/lib.rs

```diff
@@ -994,18 +994,21 @@ mod tests {
 				false,
 				keystore,
 				sp_consensus::AlwaysCanAuthor,
-			).expect("Starts aura");
+			)
+				.expect("Starts aura")
+				.unit_error()
+				.compat();
 
 			runtime.spawn(aura);
 		}
 
-		runtime.spawn(futures01::future::poll_fn(move || {
+		runtime.spawn(futures::future::poll_fn(move |_| {
 			net.lock().poll();
```

Comment:
I don't think that's correct. The call to `poll()` here expects a futures 0.1 task available and will likely panic here?

Reply:
Well, the tests pass, so I'm not sure.